tests/ci/cdk/cdk/bm_framework_stack.py

```diff
@@ -0,0 +1,147 @@
+    """Define a stack used to create a CodeBuild instance on which to execute the AWS-LC benchmarking framework"""
+
+    def __init__(self,
+                 scope: core.Construct,
+                 id: str,
+                 ecr_repo_name: str,
+                 spec_file_path: str,
+                 **kwargs) -> None:
+        super().__init__(scope, id, **kwargs)
+
+        # Define some variables that will be commonly used
+        userid = boto3.client('sts').get_caller_identity().get('Account')
+        S3_PROD_BUCKET = "{}-{}-prod-bucket".format(userid, id)
+        S3_PR_BUCKET = "{}-{}-pr-bucket".format(userid, id)
+        CLOUDWATCH_LOGS = "{}-{}-cw-logs".format(userid, id)
+
+        # Define CodeBuild resource.
+        git_hub_source = codebuild.Source.git_hub(
+            owner=GITHUB_REPO_OWNER,
+            repo=GITHUB_REPO_NAME,
+            webhook=True,
+            webhook_filters=[
+                codebuild.FilterGroup.in_event_of(
+                    codebuild.EventAction.PULL_REQUEST_CREATED,
+                    codebuild.EventAction.PULL_REQUEST_UPDATED,
+                    codebuild.EventAction.PULL_REQUEST_REOPENED)
+            ],
+            clone_depth=1)
+
+        # Define a IAM role for this stack.
+        code_build_batch_policy = iam.PolicyDocument.from_json(code_build_batch_policy_in_json([id]))
+        ec2_bm_framework_policy = iam.PolicyDocument.from_json(ec2_bm_framework_policies_in_json())
+        ssm_bm_framework_policy = iam.PolicyDocument.from_json(ssm_bm_framework_policies_in_json())
+        s3_bm_framework_policy_prod_bucket = iam.PolicyDocument.from_json(s3_bm_framework_policies_in_json(S3_PROD_BUCKET))
+        s3_bm_framework_policy_pr_bucket = iam.PolicyDocument.from_json(s3_bm_framework_policies_in_json(S3_PR_BUCKET))
+        codebuild_inline_policies = {"code_build_batch_policy": code_build_batch_policy,
+                                     "ec2_bm_framework_policy": ec2_bm_framework_policy,
+                                     "ssm_bm_framework_policy": ssm_bm_framework_policy,
+                                     "s3_bm_framework_policy_prod_bucket": s3_bm_framework_policy_prod_bucket,
+                                     "s3_bm_framework_policy_pr_bucket": s3_bm_framework_policy_pr_bucket}
+        codebuild_role = iam.Role(scope=self,
+                                  id="{}-codebuild-role".format(id),
+                                  assumed_by=iam.ServicePrincipal("codebuild.amazonaws.com"),
+                                  inline_policies=codebuild_inline_policies,
+                                  managed_policies=[
+                                      iam.ManagedPolicy.from_aws_managed_policy_name("CloudWatchAgentServerPolicy")
+                                  ])
+
+        # Create build spec.
+        placeholder_map = {"ECR_REPO_PLACEHOLDER": ecr_arn(ecr_repo_name)}
+        build_spec_content = YmlLoader.load(spec_file_path, placeholder_map)
```

Comment:
Nit: `ECR_REPO_PLACEHOLDER` should be changed to `x86_*` because this CodeBuild supports both x86 and aarch benchmark.

Reply:
Do you mean:
`placeholder_map = {"x86_ECR_REPO_PLACEHOLDER": ecr_arn(ecr_repo_name)}`
or
`placeholder_map = {"x86_*": ecr_arn(ecr_repo_name)}` ?

Reply:
Yes.

Reply:
See this [comment](https://github.com/awslabs/aws-lc/pull/209/files#r683064278). This code is replacing that the occurrence of `ECR_REPO_PLACEHOLDER` in `bm_framework_omnibus.yaml` with the correct repo name. Currently it is replacing it with `aws-lc-ecr-linux-x86`, which is what we want as the codebuild itself runs on x86. I don't see a reason for the change anymore?

Reply:
Discussed in https://github.com/awslabs/aws-lc/pull/209#discussion_r684372956